specs/waku/v2/waku-relay.md

```diff
@@ -33,26 +34,28 @@ authors: Oskar Thorén <oskar@status.im>, Sanaz Taheri <sanaz@status.im>
 
 ## Security Requirements
 
-<!-- In this part, we analyze the security of the  `relay` protocol concerning data confidentiality, integrity, authenticity, and anonymity. This is to enable users of this protocol to make an informed decision about all the security properties that can or can not achieve by deploying a `relay` protocol.-->
+The `relay` protocol supports the following security properties under a static [Adversarial Model](#adversarial-model). Note that data confidentiality[^1], integrity[^2] and authenticity[^3] are currently considered out of the scope of `relay` protocol and must be handled by the upperlevel protocols. 
 
-- **Message Publisher Anonymity**: This property indicates that no adversarial entity can link a published `Message` to its origin i.e., the peer. Note that this feature also implies the unlinkability of the publisher to its published topic ID, this is because  `Message` contains the topic IDs as well.
+[^1]: Confidentiality indicates that an adversary should not be able to learn the data carried by the `relay` protocol.
+[^2]: Integrity indicates that the data transferred by the `relay` protocol can not be tampered with by an adversarial entity without being detected.
+[^3]: Authenticity no adversary can forge data on behalf of a targeted publsiher and make it accepted by other subscribers as if the origin is the target.
 
-- **Topic Subscriber Anonymity:** This feature stands for the inability of any adversarial entity from linking a peer to its subscribed topic IDs.
 
-- **Confidentiality**: The adversary should not be able to learn the data carried by the `relay` protocol.
--  **Integrity**: This feature indicates that the data transferred by the `relay` protocol can not be tampered with by an adversarial entity without being detected.
--  **Authenticity**: No adversary can forge data on behalf of a targeted peer and make it accepted by other peers as if the origin is the target.
-- **Spam resistant**: No adversary can flood the system with spam messages (i.e., publishing a large number of messages in a short amount of time).
+- **Publisher-Message Unlinkability**: This property indicates that no adversarial entity can link a published `Message` to its publsiher. This feature also implies the unlinkability of the publisher to its published topic ID as the `Message` embodies the topic IDs.
+
+- **Subscriber-Topic Unlinkability**: This feature stands for the inability of any adversarial entity from linking a subscriber to its subscribed topic IDs.
```

Comment:
Following our last meeting, since there is only one default topic id (pubsub topic) in waku v2, such un-linkability is guaranteed by default (it provides k-anonymity where k is the size of the system). This is detailed in the security analysis as well.

Reply:
Note that one topic applies to _Waku v2_ spec, not to _Waku Relay_ spec.

Reply:
>  only one default topic id (pubsub topic) in waku v2

This is how Waku v2 is using it, but it isn't what the relay protocol provides. The relay protocol enables the use of many topics, but our recommended usage in Waku v2 spec is to use a single one.